hs_core/page_processors.py

```diff
@@ -122,7 +122,9 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     content_model.non_preferred_path_names = []
     # whether the user has permission to change the model
     can_change = content_model.can_change(request)
-    if can_change and not content_model.raccess.published:
+    if can_change and not content_model.raccess.published_or_review_pending:
+        # display of non-preferred paths is relevant for resource that is not yet published or not under review for
```

Comment:
It seems to me that this display may still be relevant for resources under review, because one review consideration could be to avoid non preferred file names.  I think that if a resource holding non preferred file names is submitted for publication, the human in the loop reviewer should be made aware of this and encourage the creators to fix, or at least justify why the file names should not be fixed.  @pkdash @devincowan

Reply:
@dtarb I was not aware that part of the review for publication can also include data files. @devincowan Are we allowing the user to edit data files when the resource is under review? You may need to make that change as part of the HIL improvements PR if it is not already doing that. I am going to rollback my last commit in this PR.

Reply:
@pkdash when the res is submitted for HIL review, we set immutable
https://github.com/hydroshare/hydroshare/blob/rc-2.1/hs_core/hydroshare/resource.py#L1050

Reply:
@devincowan That means the resource is not editable while under review. I  assume the resource will be available for edit if the review is rejected.

Reply:
@pkdash correct resource files aren't editable while under HIL review. Other metadata are editable (eg keywords, spatial, abstract) but not title etc. Same list as for published resources